CHANGES_NEXT_RELEASE

```diff
@@ -10,3 +10,4 @@
     -  Replace var with let/const
     -  Fix or disable eslint errors
 - Upgrade logops dep from 2.1.0 to 2.1.2 due to colors dependency corruption
+- Return 500 when DB is not connected
```

Comment:
```suggestion
- Return 500 when DB is not connected (#570)
```

Issue number should be mentioned in CHANGES_NEXT_RELEASE entries, whenever there is some associated.

Reply:
@fgalan , Done in commit https://github.com/telefonicaid/fiware-sth-comet/pull/574/commits/07de031f9f0c8e67071ce8dbf460323d8019fae3 below. Thanks :)